src/main/java/com/trackit/trackitDTO.java

```diff
@@ -15,10 +15,10 @@ public void setFoodName(String foodName) {
 		this.foodName = foodName;
 	}
 	public int getCalories() {
-		return Calories;
+		return calories;
 	}
 	public void setCalories(int calories) {
-		Calories = calories;
+		this.calories = calories;

```

Comment:
"this" keyword should be used to reference class attributes like in other setters.